opensearch_dashboards.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "searchRelevance",
+  "version": "2.3.0.0",
+  "opensearchDashboardsVersion": "2.3.0",
```

Comment:
This should also be updated to 3.0.0, and then 2.4.0 when the 2.x branch is cut.